templates/CRM/Activity/Form/Selector.tpl

```diff
@@ -21,10 +21,12 @@
        {/if}
        {foreach from=$columnHeaders item=header}
           <th scope="col">
-          {if $header.sort}
+          {if isset($header.sort)}
             {assign var='key' value=$header.sort}
+            {if !empty($sort)}
             {$sort->_response.$key.link}
```

Comment:
This change was a bit hard to read in the UI but I worked through it in the IDE and I think it makes sense. Not a show-stopped given the state of our smarty but with the extra if around the line above it would ideally be indented 2 spaces

Reply:
indented